### PR TITLE
feat: GEOT-765: background image more visible

### PR DESCRIPTION
**Pull request by ra8ga**

### CODE REVIEW

File: packages/app-andrew/src/components/group/GroupHeader/styles.scss [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
+@import '~@gtms/styles/scss/mixins/imageCover.scss';
 
 .groupHeader {
   display: flex;
-  padding: 8px;
+  gap: 8px;
   max-width: $tempWidth;
   border-radius: $borderRadius2;
-  background: $theme1;
+
```
Comment (ra8ga) [inline packages/app-andrew/src/components/group/GroupHeader/styles.scss:9]: I will resolve it in next PR or create a ticket

File: packages/app-andrew/src/components/group/Sidebar/styles.scss [hunk 2]
```diff
@@ -8,18 +8,21 @@
   display: flex;
   grid-gap: 8px;
 
-  .groupAvatar {
-    width: 40px;
-    height: 40px;
+  .avatarAndName {
+    // temporary hidden
```
Comment (ra8ga) [inline packages/app-andrew/src/components/group/Sidebar/styles.scss:12]: will be removed in next pr or create a ticket

File: packages/app-andrew/src/components/search/page/index.tsx [hunk 3]
```diff
@@ -438,11 +438,7 @@ export const SearchPage: FC<{
   return (
     <div className={styles.pageWrapper} data-testid="search-page">
       <div className={styles.wrapper}>
-        {/* @todo add text header instead of text on the image below */}
-        <h2 className={styles.header}>
-          {t('searchHeader1')}
-          <span>{t('searchHeader2')}</span>
-        </h2>
+        <span>{t('searchHeader2')}</span>
```
Comment (ra8ga) [inline packages/app-andrew/src/components/search/page/index.tsx:441]: will be removed in next pr or create a ticket

File: packages/app-andrew/src/pages/_app.tsx [hunk 4]
```diff
@@ -109,23 +109,30 @@ class GTMSApp extends App<GTMSAppProps, {}, GTMSAppState> {
...
+                    backgroundImage: `url(${
+                      backgroundLoaded
+                        ? backgroundImage.full
+                        : backgroundImage.mini
+                    })`,
+                  }
+                : undefined
+            }
+          />
+        </div> */}
```
Comment (ra8ga) [inline packages/app-andrew/src/pages/_app.tsx:130]: will be removed in next pr or create a ticket

File: packages/app-andrew/src/pages/_app.tsx [hunk 5]
```diff
@@ -109,23 +109,30 @@ class GTMSApp extends App<GTMSAppProps, {}, GTMSAppState> {
...
-                      ? backgroundImage.full
-                      : backgroundImage.mini
-                  })`,
-                }
-              : undefined
-          }
+          className={styles.aaaa}
+          style={{
+            backgroundImage: `url('/images/temp-images/group_bg_3.png')`,
+          }}
```
Comment (ra8ga) [inline packages/app-andrew/src/pages/_app.tsx:135]: will be removed in next pr or create a ticket

File: packages/app-andrew/src/pages/_app.tsx (hunk 4 above)
Comment (mariusz-kabala) [reply packages/app-andrew/src/pages/_app.tsx:130]: this needs to be restored. creating a ticket is not enough

File: packages/app-andrew/src/pages/appStyles.scss [hunk 6]
```diff
@@ -2,21 +2,43 @@
...
+  height: 100%;
+  width: calc(100% - 795px);
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-attachment: fixed;
+
+  width: 100%;
+  // filter: blur(55px);
+}
```
Comment (ra8ga) [inline packages/app-andrew/src/pages/appStyles.scss:44]: mocked background with proper blur - I will resolve it in next PR or create a ticket

File: packages/app-andrew/src/pages/group/[...params].tsx [hunk 7]
```diff
@@ -299,144 +294,161 @@ const GroupPage: NextPage<GroupPageProps> = (props) => {
...
-                        jeszcze żadnego posta... Możesz być pierwszy!
-                      </h3>
-                      <PostCreate groupId={state.group?.id || ''} />
-                    </div>
-                  )}
-                {state.posts.posts.length > 0 && (
-                  <div className={styles.posts}>
-                    {state.posts.isLoading && (
-                      <Spinner additionalStyles={styles.loading} />
+            <>
```
Comment (ra8ga) [inline packages/app-andrew/src/pages/group/[...params].tsx:297]: here I just added <> and wrapper with {!state.showPromoted && !state.showUsers &&  
<GroupCover

**Review by mariusz-kabala — APPROVED** [review]

**Pull request merged**